src/content/en/2022/security.md

```diff
@@ -737,7 +739,9 @@ To mitigate these concerns the HTML specification (version 5) introduced the `sa
 
 The above chart of the 2022 data shows that more than 99% of websites with a `sandbox` attribute enable the `allow-scripts` and `allow-same-origin` permissions.
 
-Of desktop websites that embed an iframe, 35.2% also include the `sandbox` attribute.
+Of desktop websites that embed an iframe, 21.2% also include the `sandbox` attribute.
```

Comment:
- That is the mobile number? The desktop number is 22.07%?
- The wording here is confusing. The percentage is not of websites but of iframes 🤔 Maybe simply add `of iframes` in front of `also` to make it clear?

Reply:
That's supposed to be the desktop number. Maybe I misinterpreted your previous comment, but I thought it to be this figure because of:

> For 2022, the numbers change from 18.9% of 11.5 million frames in mobile to 12.6% of 17.4 million frames in mobile and desktop websites that embed an iframe, 35.2% also include the sandbox attribute to 21.2%

Checking the data, it should be 22.08%, right?

I can change it to:
> For all iframes found on desktop websites, 21.08% include the `sandbox` attribute.

Reply:
Sorry about the confusion, in the initial comment above from me, I probably mixed up the lines.

Right, when rounding correctly it should be `22.08%`
The new wording is less confusing.